library/src/main/java/com/schibsted/spain/barista/custom/PerformClickAction.java

```diff
@@ -32,7 +33,11 @@ public String getDescription() {
 
       @Override
       public void perform(UiController uiController, View view) {
-        view.performClick();
+        if (view.isClickable()) {
+          view.performClick();
+        } else {
+          click().perform(uiController, view);
+        }
```

Comment:
Maybe if we extract the line ` click().perform(uiController, view);` to a method called `propagateClickToChildren()`?